recipe/conda_build_config.yaml

```diff
@@ -2,4 +2,9 @@ arm_variant_type_default:
   - sbsa
 arm_variant_type:
   - sbsa
-  - tegra
+  # arm-variant * tegra requires glibc version which is currently not available
+  # binary glibc 2.29.0 <= recipe glibc 2.28 $PREFIX/bin/nvcc
+  # The binary is not compatible with the recipe's glibc pinning.
+  # See https://docs.nvidia.com/cuda/cuda-installation-guide-linux/#system-requirements
+  # Enable this once glibc is available on the main channel.
```

Comment:
The comment above is clear but these four lines are less clear to me.. would you be able to clarify?

Reply:
I rephrased it for clarity